ns/index.html

```diff
@@ -187,6 +187,41 @@ <h5>Property Definitions</h5>
 </dd>
 </dl>
 </dd>
+<dt id="vocab_list_iterate">
+<strong>list:iterate</strong>
+<em>iterate</em>
+<span class="permalink"><a href="#vocab_list_iterate" title="Permalink for list:iterate">(link)</a></span>
+</dt>
+<dd>
+<p>Iterates over index/value pairs of the subject list</p>
+
+<p><strong>schema</strong>:
+<code>($a_1 .. $a_n) list:iterate ($i $v)</code></p>
+
+<p><strong>definition</strong>:
+Gets the matching pair of list index and list value for every member of the subject.
+If the object is a variable, it will create a solution for each member of the subject list.
+If any member of the object list is a variable it will create a solution for all matching
+members of the subject list.
+If the object is ground, it evaluates to <code>true</code>, otherwise, <code>false</code>.</p>
```

Comment:
Yes, it should say "If the object is ground, it evaluates to `true` _if the entry at the specified index matches the specified value_, otherwise `false`."